.github/workflows/build_and_test.yml

```diff
@@ -48,6 +48,11 @@ jobs:
   configure-jobs:
     name: Configure jobs
     runs-on: ubuntu-20.04
+    # All other jobs in this workflow depend on this job,
+    # so the entire workflow is skipped when these conditions evaluate to false:
+    # Run all jobs for Apache Spark repository
+    # Run only non-scheduled jobs for forked repositories
```

Comment:
Good notes, but we better to still keep below notes to mention fork/upstream trigger scopre

Reply:
I have adjusted the comments and the one condition.